doc/conf.py

```diff
@@ -61,18 +70,9 @@ def _check_deps():
             "The following dependencies are missing to build the "
             "documentation: {}".format(", ".join(missing)))
 
-
 _check_deps()
 
-try:
-    from unittest.mock import MagicMock
-except ImportError:
-    from mock import MagicMock
-
-
-# Use IPython's console highlighting by default
-extensions.extend(['IPython.sphinxext.ipython_console_highlighting',
-                   'IPython.sphinxext.ipython_directive'])
+from sphinx_gallery.sorting import ExplicitOrder
```

Comment:
Why move the import from the head of the file here?

Reply:
https://github.com/matplotlib/matplotlib/pull/9708#issuecomment-356152800

Reply:
Thanks. IMO this could use a short comment.

Reply:
done